internal/backend/local/local.go

```diff
@@ -205,7 +213,7 @@ func (b *Local) Remove(ctx context.Context, h restic.Handle) error {
 
 	// reset read-only flag
 	err := fs.Chmod(fn, 0666)
```

Comment:
I really don't get this one, though, at least not on Unix. Why chmod a file that we're going to remove right after?

Reply:
I agree that on Unix this `chmod` is pretty much useless. My idea here was to ensure that a Windows system can remove a file in a repository created on a Unix system (probably via a shared NTFS partition).